### PR TITLE
Added quiz in microprocessors.md

### PR DESCRIPTION
**Pull request by Samridhi-98**
Added pop quiz in microprocessors.md

Fixes #505 

#### Changes done:
- [x] Task 1
MCQ for `microprocessor.md` 

#### Screenshots
cvs

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@Shivansh2407 please review my PR

File: docs/seq-lsi/microprocessors.md [hunk 1]
```diff
@@ -21,8 +21,29 @@ has_children: false
...
 
 Other combinational and sequential components, such as MUXs, registers and counters are also part of the microprocessor to provide the functionality required by the software, to optimise its execution and to connect to other external components.
 
 One of the first "single-chip" microprocessors is the Intel 4004. Read more about it and other interesting microprocessors and their history in the [IEEE Chip Hall of Fame](https://spectrum.ieee.org/static/chip-hall-of-fame) and the website [Great Microprocessors of the Past and Present](http://www.cpushack.com/CPU/cpu.html) at the CPUSHACK museum.
+
+{:.quiz}
+
+1. Which of the following is `single-clip` microprocessors?.
```
Comment (manjotsidhu) [inline docs/seq-lsi/microprocessors.md:32]: Space before question is required at all occurrences
```suggestion
1. Which of the following is `single-clip` microprocessors ?
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]
Please remove the unnecessary changes (newlines) in other files. Let's keep PRs neat and tidy.

**Comment by Samridhi-98** [comment]
@Shivansh2407  Will you review my PR.

**Review by Shivansh2407 — APPROVED** [review]
Great Work @Samridhi-98 🎉 I see a lot of file changes here, merging this but make sure no extra spaces are added in any files when you make the PR

**Pull request merged**